assets/install.sh

```
@@ -118,7 +118,7 @@ main() {
 
     # Create temp directory
     local tmp_dir=$(mktemp -d)
-    local tmp_file="${tmp_dir}/gtx-cli"
+    local tmp_file="${tmp_dir}/gt"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)